docs/UserGuide.md

```diff
@@ -118,31 +118,54 @@ Upon completing this section, you should be able to explore SoConnect on your ow
+The [parameters](#622-parameter) required for each feature can be found in [Command Summary](#9-command-summary) and [Commands](#7-commands).
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+<div markdown="block" class="alert alert-info"> 
+
+**:information_source: Note:**<br>
+
+* Words in `UPPER_CASE` are the information to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is an information which can be used as `add n/John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family`, etc.
 
-* Parameters can be in any order unless explicitly stated otherwise.<br>
+* [Parameters](#622-parameter) can be in any order unless explicitly stated otherwise.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+* If a [parameter](#622-parameter) is expected only once in the [command](#621-command), but you specified it multiple times, only the last occurrence of the [parameter](#622-parameter) will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+* Irrelevant [parameters](#622-parameter) for [commands](#621-command) that do not take in [parameters](#622-parameter) (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-* Tasks will be referred to as Todos
-
-* `INDEX` is used in commands to refer to a specific contact or todo by their index number on the currently displayed contact list and todo list. The `INDEX` **must be a positive non-zero integer** 1, 2, 3, …​ <a id="command-format-index"></a>
+* `INDEX` is used in [commands](#621-command) to refer to a specific contact or todo by their index number on the currently displayed contact list and todo list. The `INDEX` **must be a positive non-zero integer** 1, 2, 3, …​ <a id="command-format-index"></a>
 
 ### 6.4. How to Use the CLI
 
+If you already know about [Command Line Interface (CLI)](#CLI), you can skip this part and read the [Commands](#7-commands).
+
+1. To use the [Command Line Interface (CLI)](#CLI), you can type a [command](#621-command), which is supposed to be typed in a certain [format](#63-command-format), in the command box and press Enter. SoConnect will perform a specific action. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```

Comment:
I think this section is too blurry and rushed. Perhaps can try to illustrate with more complex command, such as adding people. Can say about:

Command format for add:

Short explanation about the command format, what is optional and what is not, what is multiple times etc.

Show examples of acceptable and unacceptable commands, and their mistakes.

This can be done after pr merging, can merge first.

Reply:
Ok